store_queue.py

```diff
@@ -0,0 +1,55 @@
+import pika
+import settings
+
+
+class StoreQueue(object):
+
+    def __init__(self, logger):
+        self._logger = logger
+        self._connection = None
+        self._channel = None
+
+    def __connect(self):
+        self._logger.debug("Connecting to queue")
+        for url in settings.RABBIT_URLS:
+            try:
+                self._connection = pika.BlockingConnection(pika.URLParameters(url))
+                self._channel = self._connection.channel()
+                self._channel.queue_declare(queue=settings.RABBIT_QUEUE)
+                self._logger.debug("Connected to queue", url=url)
+                return True
+
+            except pika.exceptions.AMQPConnectionError as e:
+                self._logger.error("Unable to connect to queue", exception=repr(e), url=url)
+                continue
+
+        return False
+
+    def __disconnect(self):
+        try:
+            self._connection.close()
+            self._logger.debug("Disconnected from queue")
+
+        except Exception as e:
+            self._logger.error("Unable to close connection", exception=repr(e))
+
+    def __publish(self, notification):
```

Comment:
If we're trying to make this class a generic queue publisher it would be better to rename notification to message